File: BI-TJV/semestral/rest-server/src/main/java/wrzecond/repository/TjvEmployeeRepository.java
package wrzecond.repository;

import org.springframework.data.jpa.repository.Query;
import org.springframework.stereotype.Repository;
import wrzecond.entity.TjvEmployee;

@Repository
public interface TjvEmployeeRepository extends TjvRepository<TjvEmployee> {

    @Query("SELECT e FROM TjvEmployee e WHERE e.username = :username AND e.password = :password")
    TjvEmployee getByUsernamePassword (String username, String password);

}

File: BI-KOT/semestral/rest-server/src/main/kotlin/wrzecond/repository/TjvEmployeeRepository.kt
package wrzecond.repository

import org.springframework.data.jpa.repository.Query
import org.springframework.stereotype.Repository
import wrzecond.entity.TjvEmployee

@Repository
interface TjvEmployeeRepository : TjvRepository<TjvEmployee> {

    @Query("SELECT e FROM TjvEmployee e WHERE e.username = :username AND e.password = :password")
    fun getByUsernamePassword (username: String, password: String) : TjvEmployee

}
